diff --git a/donut/nvrhi/thirdparty/Vulkan-Headers/include/vulkan/vulkan.hpp b/donut/nvrhi/thirdparty/Vulkan-Headers/include/vulkan/vulkan.hpp
index e20d00f..87392ed 100644
--- a/donut/nvrhi/thirdparty/Vulkan-Headers/include/vulkan/vulkan.hpp
+++ b/donut/nvrhi/thirdparty/Vulkan-Headers/include/vulkan/vulkan.hpp
@@ -17482,7 +17482,7 @@ namespace VULKAN_HPP_NAMESPACE
         else
         {
 #  if defined( _WIN32 )
-          m_library = ::LoadLibraryA( "vulkan-1.dll" );
+          m_library = ::LoadLibraryA( "sl.interposer.dll" );
 #  elif defined( __APPLE__ )
           m_library = dlopen( "libvulkan.dylib", RTLD_NOW | RTLD_LOCAL );
           if ( !m_library )
